### DISCUSSION TITLE
Does AR.js support particles?

### DISCUSSION THREAD

**Discussion by hermionewy** [AR-js-org/AR.js#261] [Q&A] [answered]
I'm hoping to combine data with WebAR to create a project, with each particle representing one data point. I could not find any examples so far, so I wonder if AR.js support putting, say 15k particles, in the air? Is it possible to focus/click/point on a particle for 3 seconds and trigger a tooltip?

**Comment by jonnypage** [comment, marked as answer]
You can incorporate particles using the [aframe-particle-system-component](https://github.com/IdeaSpaceVR/aframe-particle-system-component)

However, I don't think you can do a hittest on particles. If you wanted to do that, you'd have to use a-frame primatives such as `<a-plane>`, position and and apply a texture to them, then perform the hittest on the primative.
